### PR TITLE
Update UG for consult features

### PR DESCRIPTION
**Pull request by wesho1107**
Add User Guide for features on consultations.

### CODE REVIEW

**Review by Originalidk — APPROVED** [review]
LGTM.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -251,7 +251,89 @@ Examples:
...
+* TIME The time of consultation
+* STUDENT_NAME The name of the student(s)
+
+Examples:
+* `addconsult d/2023-10-30 t/12:30 n/John Doe` creates a consultation for John Doe on 2023-10-30 12:30.
+* `addconsult d/2023-09-30 t/15:30 n/John Doe n/Foo Bar n/Rayson n/Wesley` creates a consultation for John Doe, Foo Bar
+  , Rayson and Wesley on 2023-09-30 15:30.
+
+
+### Showing list of consultations: `showconsults`
```
Comment (raydenlim) [inline docs/UserGuide.md:270]: showconsults and viewconsults may sounds very similar.
Perhaps, you could change showConsults to listConsults, since it lists down all upcoming consultations

File: docs/UserGuide.md [hunk 2]
```diff
@@ -251,7 +251,89 @@ Examples:
 * `deletetask 2` deletes the task with index 2 from the task list.
 
 
+### Creating a Consult: `createconsult`
+Creates a consultation into upcoming consultations.
```
Comment (raydenlim) [inline docs/UserGuide.md:255]: 'Creates a consultation into upcoming consultations.'
This sounds confusing, might want to rephrase it to:
Creates a new consultation.

Becareful of the naming as well. Line 254, you used 'createconsult', 
those below (i.e Line 265, 266)you used 'addconsult'

File: docs/UserGuide.md [hunk 3]
```diff
@@ -251,7 +251,89 @@ Examples:
...
+
+
+### Remove students from a consultation: `removefromconsult`
+Removes student(s) from a consultation.
 
+Format: `removefromconsult INDEX n/STUDENT_NAME …`
+
+Parameters:
+* INDEX The index of the consultation in the upcoming consultations.
+* STUDENT_NAME The name of student to be added into the consultation.
```
Comment (raydenlim) [inline docs/UserGuide.md:319]: ** STUDENT_NAME The name of student to be **removed** from the consultation

**Review by raydenlim — CHANGES_REQUESTED** [review]

**Comment by raydenlim** [comment]
Woops, merged alr, but rmb to update accordingly ;D
